src/frontend/Sider/Behandling/Brev/DelmalMeny.tsx

```diff
@@ -50,7 +61,24 @@ export const DelmalMeny: React.FC<Props> = ({ delmal, settValgfelt, variabler, s
                             />
                         );
                     case 'fritekst':
-                        return <Fritekst key={index} />;
+                        return (
+                            <Fritekst
+                                key={index}
+                                avsnitt={fritekst[delmal._id] || [lagTomtAvsnitt()]}
+                                settAvsnitt={(utledNextState) => {
+                                    const prevState = fritekst[delmal._id] || [lagTomtAvsnitt()];
+                                    const nextState =
+                                        typeof utledNextState === 'function'
+                                            ? utledNextState(prevState)
+                                            : utledNextState;
+
+                                    settFritekst((prevState) => ({
+                                        ...prevState,
+                                        [delmal._id]: nextState,
+                                    }));
+                                }}
```

Comment:
Dette ble kanskje i overkant komplisert, måtte gjøre det for å håndtere mulig undefined inittate. På denne måten har et fritekstavsnitt heller ikke tilgang til å endre andre avsnitt.

Men mulig man burde trekke ut funksjonen for å bruke den både i valgfelt og her.